EasyCommands/BlockHandlers/BlockHandlers.cs

```diff
@@ -76,23 +76,25 @@ public static class BlockHandlerRegistry {
                 return blockHandlers[blockType];
             }
 
-            public static List<Object> GetBlocks(Block blockType, Func<IMyTerminalBlock, bool> selector = null) {
-                if (PROGRAM.invalidBlockCache) {
+            public static List<Object> GetSelf(Block? blockType) =>
+                blockType == null || blockType == Block.DISPLAY
+                    ? blockHandlers[blockType ?? Block.PROGRAM].SelectBlocks(NewList(PROGRAM.Me))
+                    : null;
```

Comment:
This is breaking the behavior of SelfSelector, which should return all blocks of the given type (no selector name) if the block type is anything but Display, Program or null.

Reply:
the `SelfSelector` calls `BGR.GetSelf(blockType) ?? BGR.GetBlocks(GetBlockType())`